.github/workflows/upload-assets.yml

```diff
@@ -11,16 +11,16 @@ jobs:
       matrix:
         build: [linux, macos, windows-gnu, windows-msvc]
         include:
-          - build: linux
+          - build: linux-x86_64
```

Comment:
I believe that updating this would require the corresponding values in `build` so that the matrix legs will still match.

Reply:
Will do. I'll push it as a separate commit but if you want me to rebase it into one I can do that too.

Reply:
Great thanks! And no worries. In cases like this where the diff is so small it's fine to squash the changes on the original commit, but will just squash on merge